src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -214,16 +214,24 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
 
   /** TODO: internal */
   _onDragStart() {
-    this._slideRenderer.startThumbDrag(this.checked);
+    if (!this.disabled) {
+      this._slideRenderer.startThumbDrag(this.checked);
+    }

```

Comment:
I actually planned to add some gesture tests in another PR (because of big changes), but I will try to add them to this PR now :)